utils/tree_utils.py

```diff
@@ -29,7 +29,7 @@ def merge_lines(lines_1, lines_2):
             """
             lines = []
             for l1, l2 in zip(lines_1, lines_2):
-                lines.append(l1 + l2)
+                lines.append(l1 + str(l2))
```

Comment:
otherwise can't print tree for uint8 alphabet

Reply:
thanks for the fix